pygmt/src/coast.py

```diff
@@ -66,9 +66,8 @@ def coast(self, **kwargs):
     {J}
     {R}
     area_thresh : int, float, or str
-        *min_area*\ [/*min_level*/*max_level*][**+ag**\|\ **i**\
-        \|\ **s**\|\ **S**][**+r**\|\ **l**][**+p**\
-        *percent*].
+        *min_area*\ [/*min_level*/*max_level*][**+a**\[**g**\|\ **i**]
+        [**s**\|\ **S**][**+l**\|\ **r**][**+p**\ *percent*].
         Features with an area smaller than min_area in km^2 or of
```

Comment:
```suggestion
        Features with an area smaller than min_area in km<sup>2</sup> or of
```
Not sure if the html tags will work, but if they do, I think it looks better than "km^2"

Reply:
Thanks, edited it using https://docutils.sourceforge.io/docs/ref/rst/roles.html#superscript